### PR TITLE
[Navbar,ArtistShow] Applies Flex to widths/heights

### PR DESCRIPTION
**Pull request by imartinez921**
Converts widths/heights originally using _calc_ into Flexbox for more responsive design.
Fixes bug where ArtistShow background was extending behind the footer area.
Applies Prettier formatting.

https://github.com/imartinez921/versify_full-stack/assets/102888592/074c5097-a34a-414c-80ef-e648b92802b2

### CODE REVIEW

**Review by k-shan — CHANGES_REQUESTED** [review]
See if you can pull out the artist style that isn't dynamic (ie. not dependent on the variable)

File: frontend/components/artists/artist_show.jsx [hunk 1]
```diff
@@ -4,64 +4,65 @@ import ArtistHeader from "./artist_header";
...
-                </div>
-                <div className="artist-nav">
+	const artistShow = (
+		<>
+			<div
+				className="artist-show-background"
+				style={{
+					backgroundImage: `url(${currentArtist.photoUrl})`,
+					backgroundSize: "50%",
+					height: "100%",
```
[the remark this answers is not in the document]
Comment (imartinez921) [reply frontend/components/artists/artist_show.jsx:37]: I can try. Thanks.

[the remark this answers is not in the document]
Comment (imartinez921) [reply frontend/components/artists/artist_show.jsx]: Done

File: frontend/components/artists/artist_show.jsx [hunk 2]
```diff
@@ -4,64 +4,65 @@ import ArtistHeader from "./artist_header";
...
+						<ArtistHeader currentArtist={currentArtist} />
+					</div>
+					<div className="artist-nav"></div>
+					{albums.length > 0 ? (
+						<AlbumIndex
+							albums={albums}
+							history={history}
+							displayAlbum={displayAlbum}
+						/>
+					) : null}
```
[the remark this answers is not in the document]
Comment (imartinez921) [reply frontend/components/artists/artist_show.jsx]: Ty

**Review by k-shan — APPROVED** [review]

**Pull request merged**